dod-wrapper/gradle.properties

```diff
@@ -0,0 +1,5 @@
+POM_ARTIFACT_ID=dod-wrapper
+VERSION_NAME=1.5.5
```

Comment:
let's maybe up all versions to 1.5.6, GitHub releases are weird with multiartifact publishing, that way we'll have one single 1.5.6 release with a new module available

Reply:
Done 😁